main.tf

```diff
@@ -2,4 +2,21 @@ module "sagemaker" {
   source                          = "./modules/sagemaker"
   model_name                      = var.model_name
   sagemaker_image_repository_name = var.sagemaker_image_repository_name
-}
\ No newline at end of file
+  tags                            = var.tags
+  sagemaker_execution_role_arn    = module.iam.sagemaker_role_arn
```

Comment:
Are you able to make this into sagemaker_execution_role_arn = var.sagemaker_execution_role_arn?

Reply:
I can, but the idea is to create the necessary resources, like sagemaker_execution_role, within AWS-MLOps-module so any repos that use it won't have to pass such resources to it

Reply:
Ah okay, yes that makes sense